governator-test-junit/build.gradle

```diff
@@ -1,6 +1,11 @@
 apply plugin: 'java'
 
 dependencies {
-    compile     project(':governator')
-    compile     "junit:junit:4.11"
+    compile    project(':governator')
+    compile    project(':governator-api')
+    compile    project(':governator-core')
+    compile    'com.netflix.archaius:archaius2-guice:2.0.6'
+    compile    'com.netflix.archaius:archaius2-test:2.0.6'
+    compile    'junit:junit:4.12'
```

Comment:
There isn't a junit version set globally, and junit on the inherited projects (core, api, legacy) is only in testCompile so it isn't coming into this project transitively. 

In the case of this project, its a test utility and its junit version will be brought to consumers transitively, so I went with 4.12 as it is the latest release for junit.